regression/cbmc-incr-oneloop/Makefile

```diff
@@ -1,10 +1,10 @@
 default: tests.log
 
 test:
-	@../test.pl -c "perl -e 'alarm shift @ARGV; exec @ARGV' 30 ../../../src/cbmc/cbmc --slice-formula"
+	@../test.pl -c "perl -e 'alarm shift @ARGV; exec @ARGV' 8 ../../../src/cbmc/cbmc --slice-formula"
```

Comment:
Why is there a `perl -e` wrapper for those tests?

Reply:
for the timeout

Reply:
I think a comment would be nice to have.